karma.conf.js

```diff
@@ -22,15 +22,15 @@ module.exports = function (config) {
     },
     customLaunchers: {
       ChromeHeadless: {
-      base: 'Chrome',
-      flags: [
-        '--headless',
-        '--disable-gpu',
-        // no-sandbox - for running under docker
-        '--no-sandbox',
-        // Without a remote debugging port, Google Chrome exits immediately.
-        '--remote-debugging-port=9222',
-        ],
+        base: 'Chrome',
+        flags: [
+          '--headless',
+          '--disable-gpu',
+          // no-sandbox - for running under docker
+          '--no-sandbox',
+          // Without a remote debugging port, Google Chrome exits immediately.
+          '--remote-debugging-port=9222'
+        ]
```

Comment:
Please don't change the indentations in the files
We should see only the functional changes you did

Reply:
Even after I remove this indentations on github, they appear again after commit.